storage/cloud-client/snippets_test.py

```diff
@@ -287,3 +294,81 @@ def test_bucket_lifecycle_management(test_bucket, capsys):
     out, _ = capsys.readouterr()
     assert "[]" in out
     assert len(list(bucket.lifecycle_rules)) == 0
+
+
+def test_create_bucket_class_location():
+    bucket = storage_create_bucket_class_location.create_bucket_class_location(
+        "storage-snippets-test"
+    )
+
+    assert bucket.location == 'US'
+    assert bucket.storage_class == "COLDLINE"
+    bucket.delete(force=True)
```

Comment:
Please use a fixture to delete these bucket, so it's always done even if the test fails or encounters another error. You can use a generator to return a UUID above clean up after the bucket it created.